localisation/.~lock.000_wc_traits.csv#

```diff
@@ -0,0 +1 @@
+Jonas Nowacki,Arithon-Desktop/Arithon,Arithon-Desktop,02.08.2019 17:55,file:///C:/Users/Arithon/AppData/Roaming/OpenOffice/4;
```

Comment:
no idea what OpenOffice temporary file do here

Reply:
I'll remove it